### DISCUSSION TITLE
Is there a way to have code block without highlighting?

### DISCUSSION THREAD

**Discussion by JosefNemec** [dotnet/docfx#8715] [General]
Just omitting language doesn't work and it looks like that docfx is trying to figure out language instead.

For example, this block:

    ```
    Toolbox.exe verify <manifest_type> <manifest_past>
    ```

will have `vim` highlighting applied in resulting generated html. Can I somehow disable this automatic language detection? Wasn't able to find anything about it in the docs.

**Comment by KalleOlaviNiemitalo** [comment]
How about ```` ```none ````

**Reply by JosefNemec** [reply]
I tried that and it removes all code block styling completely, not just syntax highlighting. Which seems to happen every time you set any unknown language.

**Comment by Crypt32** [comment]
I ran into similar issue and proposed workaround (`` ```none ``) doesn't meet requirements, because it removes styling completely. However I found a better one:
````
```text
Toolbox.exe verify <manifest_type> <manifest_past>
```
````
then it renders code block with normal style and removes code highlighting only:
![image](https://github.com/user-attachments/assets/e99c8f30-0298-4581-a966-f91016a7572b)

HTH
